### PR TITLE
MVP-460 add necessary page types

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: core/templates/core/curated_topic.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% load wagtailcore_tags %}
+
+<ol class="learn__topics-list">
+    <li class="learn__topic-item">
+        <div class="grid">
+            <div class="learn__topic-item-details c-1-3-l">
+                <h2 class="learn__topic-item-title">{{ value.title }}</h2>
+                <time class="learn__topic-duration" datatime="5M"></time>
```
Comment (richtier) [inline core/templates/core/curated_topic.html:8]: hardcoded to 5 mins on purpose?

File: learn/templates/learn/automated_list_page.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <div class="container">
+        {% breadcrumbs 'Learn how to export' %}
+            <a href="{% url 'core:dashboard' %}" id="breadcrumbs-home">Home</a>
+        {% endbreadcrumbs %}
+    </div>
+    <section id="learn-root" class="learn__page learn__categories-page container">
+        <section class="learn__category-header">
+            <div class="grid">
+                <div class="learn__introduction w-full-m c-1-2-l">
+                    <h1 class="h1">{{ page.title }}</h1>
```
Comment (richtier) [inline learn/templates/learn/automated_list_page.html:17]: does `class="h1"` work? I think `h-l` is the one you want :)

Comment (sdonk) [reply learn/templates/learn/automated_list_page.html:17]: I didn't change the HTML, just replaced the hardcoded content with variables @jestanoff is your man

File: core/templates/core/curated_topic.html (hunk 1 above)
Comment (sdonk) [reply core/templates/core/curated_topic.html:8]: Nope, mistake. I'll remove it. It's really difficult to calculate reading time, we can do it in another ticket

File: core/management/commands/create_tours.py [hunk 3]
```diff
@@ -57,5 +57,10 @@
 class Command(BaseCommand):
 
     def handle(self, *args, **options):
+        export_plan_dashboard = models.DetailPage.objects.filter().get(
```
Comment (sdonk) [inline core/management/commands/create_tours.py:60]: Odd, this is not my code

**Pull request merged**